EIPS/eip-1344.md

```diff
@@ -14,7 +14,7 @@ created: 2018-08-22
 This EIP adds an opcode that returns the current chain's EIP-155 unique identifier.
 
 ## Motivation
-[EIP-155](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-155.md) proposes to use the chain ID to prevent replay attacks between different chains. It would be a great benefit to have the same possibility inside smart contracts when handling signatures, especially for Layer 2 signature schemes using [EIP-712](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-712.md).
+[EIP-155](eip-155.md) proposes to use the chain ID to prevent replay attacks between different chains. It would be a great benefit to have the same possibility inside smart contracts when handling signatures, especially for Layer 2 signature schemes using [EIP-712](eip-712.md).
```

Comment:
The current recommendation is to use https://eips.ethereum.org/EIPs/eip-155, etc.

Reply:
I was trying to find an example but didn't from random clicking. I thought the website sutogenerstor might be able to recognize local links, but I will do this instead

Reply:
```suggestion
[EIP-155](https://eips.ethereum.org/EIPS/eip-155) proposes to use the chain ID to prevent replay attacks between different chains. It would be a great benefit to have the same possibility inside smart contracts when handling signatures, especially for Layer 2 signature schemes using [EIP-712](https://eips.ethereum.org/EIPS/eip-712).
```

Reply:
Yeah I agree, I'd like to just use file names and let jekyll sort it our for the generated page. But there seemed to be a consensus to use the link directly.